jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -114,6 +118,38 @@ public static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	private String checkPointDir;
+	private boolean keepRunning = false;
+	private boolean shellJob = true;
+	private boolean extUnixSupport = true;
+	private int logLevel = 4;
+	private String logFile = "checkpoint.log";
+
+	public CRIUSupport setCheckPointDir(String checkPointDir) {
+		this.checkPointDir = checkPointDir;
+		return this;
+	}
+	public CRIUSupport setKeepRunning(boolean keepRunning) {
+		this.keepRunning = keepRunning;
+		return this;
+	}
+	public CRIUSupport setShellJob(boolean shellJob) {
+		this.shellJob = shellJob;
+		return this;
+	}
+	public CRIUSupport setExtUnixSupport(boolean extUnixSupport) {
+		this.extUnixSupport = extUnixSupport;
+		return this;
+	}
+	public CRIUSupport setLogLevel(int logLevel) {
+		this.logLevel = logLevel;
+		return this;
+	}
+	public CRIUSupport setLogFile(String logFile) {
+		this.logFile = logFile;
+		return this;
+	}
+
```

Comment:
can you also add helpers and natives for `criu_add_inherit_fd` and `criu_set_file_locks`

Reply:
What do you mean by natives? Shouldn't I just follow the other helpers and pass in parameters through the receiver?

Reply:
Yes, I mean add the `criu_add_inherit_fd` and `criu_set_file_locks` native calls native calls to the checkpointJVMImpl native. So basically "follow the other helpers and pass in parameters through the receiver"